config/config_test.go

```diff
@@ -7,15 +7,22 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+// A number of these tests set environment variables.
+// When setting an env var, ensure that it is unset
+// at the end of the test so as not to impact other tests
+
```

Comment:
@davexroth indeed! We might need to do further refactoring if at some point we want to run this tests in parallel (which we should).